docs/guides/nextauth-migration.mdx

```diff
@@ -0,0 +1,253 @@
+---
+title: Migrating from NextAuth.js
+description: Migrating from NextAuth.js
+---
+
+This guide shows how to migrate a NextAuth.js application to Clerk for authentication.
+
+## Install `@clerk/nextjs`
+
+Clerk's Next.js SDK gives you access to prebuilt [components](https://clerk.com/docs/components/overview), [hooks](https://clerk.com/docs/references/nextjs/overview#client-side-helpers), and [helpers](https://clerk.com/docs/references/nextjs/overview) for Next.js Server Components, Route Handlers and Middleware. Install it by running the following command in your terminal:
+
+```tsx
+npm install @clerk/nextjs
+```
+
+## Set environment keys
+
+Create an account on [clerk.com](http://clerk.com) and create a new application.
+
+Store the keys in the `.env.local` file of your Next.js application.
+
+```tsx
+NEXT_PUBLIC_CLERK_PUBLISHABLE_KEY=pk_test_••••••••••••••••••••••••••••••••••
+CLERK_SECRET_KEY=sk_test_••••••••••••••••••••••••••••••••••
+```
+
+## Signing in
+
+**Account Portal**
+
+Account Portal is the fastest way to authenticate your app. It does not require you to build any authentication UI in your app, similar to the [default unbranded pages](https://next-auth.js.org/getting-started/rest-api#get-apiauthsignin) provided by NextAuth. [Read more about Account Portal](https://clerk.com/docs/account-portal/getting-started).
+
+To use the Account Portal, simply replace the links to the sign-in/sign-up pages with `<SignInButton>` or `<SignUpButton>`.
```

Comment:
```suggestion
To use the Account Portal, simply replace the links to the sign-in/sign-up pages with `[<SignInButton>](/docs/components/unstyled/sign-in-button)` or `[<SignUpButton>](/docs/components/unstyled/sign-out-button)`.
```

Reply:
You applied the links, but I think the format I wrote them in is wrong and they are not links.